crates/rolldown_binding/src/options/plugin/binding_plugin_options.rs

```diff
@@ -65,11 +65,11 @@ pub struct BindingPluginOptions {
 
   #[serde(skip_deserializing)]
   #[napi(
-    ts_type = "(ctx:  BindingTransformPluginContext, id: string, code: string) => MaybePromise<VoidNullable<BindingHookTransformOutput>>"
+    ts_type = "(ctx:  BindingTransformPluginContext, id: string, code: string, module_type: string) => MaybePromise<VoidNullable<BindingHookTransformOutput>>"
```

Comment:
```suggestion
    ts_type = "(ctx:  BindingTransformPluginContext, id: string, code: string, extraOptions: string) => MaybePromise<VoidNullable<BindingHookTransformOutput>>"
```

would be better. We might need to add more fields in it.

Reply:
We could call it module meta in js side, since it's more intutiive. In rust side, you could call it anything with a reasonable name.

Reply:
why not `TransformHookExtra` rather than `TransformHookExtraOptions` ? It is metadata not options.

Reply:
Yeah. We could use `TransformHookExtra`.  We could use names different from the js side as long as it's reasonable in rust side.

Reply:
TransformHookExtraArgs sounds more reasonable.